docs/reference/query-languages/esql/_snippets/functions/parameters/top.md

```diff
@@ -11,3 +11,6 @@
 `order`
 :   The order to calculate the top values. Either `asc` or `desc`, and defaults to `asc` if omitted.
 
+`outputField`
+:   The extra field that, if present, will be the output of the TOP call instead of `field`.{applies_to}`stack: ga 9.3.`
```

Comment:
```suggestion
:   The extra field that, if present, will be the output of the TOP call instead of `field`.{applies_to}`stack: ga 9.3`
```
The extra `.` was making this an incorrect value

Reply:
Done.